pairtools/cli/dedup.py

```diff
@@ -160,7 +160,8 @@
     "duplicates. Can be either provided as 0-based column indices or as column "
     'names (requires the "#columns" header field). The option can be provided '
     "multiple times if multiple column pairs must match. "
-    'Example: --extra-col-pair "phase1" "phase2". [output format option]',
+    'Example, to dedup phased pairs:'
```

Comment:
Suggested change: 

Example for dedup of phased pairs, remove duplicates matching in columns phase1 and (independently) in column phase2. Note how each --extra-col-pair expects a pair of columns as input: